src/System.Runtime/tests/project.json

```diff
@@ -8,6 +8,7 @@
     "System.ObjectModel": "4.0.13-beta-devapi-24424-01",
     "System.Reflection": "4.1.1-beta-devapi-24424-01",
     "System.Reflection.Emit": "4.0.2-beta-devapi-24424-01",
+    "System.Runtime": "4.2.0-beta-devapi-24424-01",
```

Comment:
I don't think you need this given that the test csproj should have a .pkgproj reference to System.Runtime already